tests/show_techsupport/tech_support_cmds.py

```diff
@@ -115,10 +115,10 @@
     "vtysh{} -c 'show bgp ipv4 labeled-unicast'",
     "vtysh{} -c 'show bgp ipv6 labeled-unicast'",
     "vtysh{} -c 'show bgp mac hash'",
-    re.compile(r'vtysh{}\s+-c "show ip bgp neighbors .* advertised-routes"'),
-    re.compile(r'vtysh{}\s+-c "show ip bgp neighbors .* routes"'),
-    re.compile(r'vtysh{}\s+-c "show bgp ipv6 neighbors .* advertised-routes"'),
-    re.compile(r'vtysh{}\s+-c "show bgp ipv6 neighbors .* routes"'),
+    re.compile(r"vtysh(\s+-Ec 'show bgp ipv4 neighbors .* advertised-routes' "
```

Comment:
does wildcard support here?

Reply:
Can you clarify your question? It does work with the changes in the sonic-utilities PR.

Reply:
no issue, I just remember this was not supported, sorry maybe I missed.

Reply:
Can you please approve if all good?